src/shared/proto/robot_component_status.proto

```diff
@@ -0,0 +1,176 @@
+syntax = "proto2";
+
+/* Data about the status of the break beam */
+message BreakBeamStatus
+{
+    // Whether or not the ball is interrupting the robot’s laser beam.
+    required bool ball_in_beam = 1;
+
+    // The reading of the robot’s laser sensor.
+    required double break_beam_reading = 2;
```

Comment:
Not sure what the range of this field is atm but IMO we should normalize it to [0, 1] and then we can remove the `break_beam_scale` since I don't think it's useful, but this may be a question for ELEC too.

I also think this field should be a `float`, since there's no need for `double` precision